.github/workflows/codeql.yml

```diff
@@ -29,25 +26,15 @@ jobs:
     # Initializes the CodeQL tools for scanning.
     - name: Initialize CodeQL
       uses: github/codeql-action/init@v1
-      # Override language selection by uncommenting this and choosing your languages
-      # with:
-      #   languages: go, javascript, csharp, python, cpp, java
-
-    # Autobuild attempts to build any compiled languages  (C/C++, C#, or Java).
-    # If this step fails, then you should remove it and run the build manually (see below).
-    #- name: Autobuild
-    #  uses: github/codeql-action/autobuild@v1
-
-    # ℹ️ Command-line programs to run using the OS shell.
-    # 📚 https://git.io/JvXDl
-
-    # ✏️ If the Autobuild fails above, remove it and uncomment the following three lines
-    #    and modify them (or add more) to build your code if your project
-    #    uses a compiled language
-
-    #- run: |
-    #   make bootstrap
-    #   make release
+      with:
+        languages: ${{ matrix.language }}
+        # If you wish to specify custom queries, you can do so here or in a config file.
+        # By default, queries listed here will override any specified in a config file.
+        # Prefix the list here with "+" to use these queries and those in the config file.
+        # queries: ./path/to/local/query, your-org/your-repo/queries@main
+
+    - name: Autobuild
```

Comment:
This is what confused me before: if Autobuild isn't required for the analysis (previous version), why do we need it explicitly set (new version)?

Reply:
@jhutchings1, I'm ready to merge this one, just wanted a clarification on this before that.

Reply:
You're right! It doesn't actually hurt either way, as it'll succeed automatically. We could patch this to remove the Autobuild, but it's not strictly necessary.

Reply:
I'm fine following the official recommendation. I actually find it clearer to have the step listed here explicitly :-)